src/allotropy/parsers/appbio_quantstudio_designandanalysis/structure/generic/structure.py

```diff
@@ -466,6 +437,14 @@ class Result:
     slope: float | None
     efficiency: float | None
 
+    @classmethod
+    def get_genotyping_determination_result(cls, _: SeriesData) -> str | None:
+        return None
```

Comment:
It seems like this can only return `None` so Im a bit confused by the str type return. Is this conforming to the method signature of some class template? Just flagging in case this class should inherit from some parent class template. Again maybe this is the existing code style so not blocking on this but definitely curious!

Reply:
This function is design to be overwritten by the experiment types that do have a genotyping determination result

the default behavior is to not have this value, that is why the function is just a return None

Reply:
Ahhh this is the base class my bad